suite/cstest/include/test_detail_bpf.h

```diff
@@ -41,6 +43,10 @@ static const cyaml_schema_field_t test_detail_bpf_op_mapping_schema[] = {
 			       TestDetailBPFOp, mem_base, 0, CYAML_UNLIMITED),
 	CYAML_FIELD_INT("mem_disp", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
```

Comment:
```suggestion
	CYAML_FIELD_UINT("mem_disp", CYAML_FLAG_OPTIONAL, TestDetailBPFOp,
```

I made the mistake there marking them as `INT` instead of `UINT`. Could you please make them `UINT` (also `imm`, `off` etc.) while you are at it.